src/azure-cli/azure/cli/command_modules/storage/operations/blob.py

```diff
@@ -21,6 +22,26 @@
 from knack.util import CLIError
 
 
+def create_container(cmd, container_name, resource_group_name=None, account_name=None,
+                     metadata=None, public_access=None, fail_on_exist=False, timeout=None,
+                     default_encryption_scope=None, deny_encryption_scope_override=None, **kwargs):
+    if default_encryption_scope is not None or deny_encryption_scope_override is not None:
+        from .._client_factory import storage_client_factory
+        client = storage_client_factory(cmd.cli_ctx).blob_containers
+        BlobContainer = cmd.get_models('BlobContainer', resource_type=ResourceType.MGMT_STORAGE)
+        blob_container = BlobContainer(default_encryption_scope=default_encryption_scope,
+                                       deny_encryption_scope_override=deny_encryption_scope_override)
+        container = client.create(resource_group_name=resource_group_name, account_name=account_name,
+                                  container_name=container_name, blob_container=blob_container)
+        return container is not None
+
+    from .._client_factory import blob_data_service_factory
+    kwargs['account_name'] = account_name
+    client = blob_data_service_factory(cmd.cli_ctx, kwargs)
+    return client.create_container(container_name, metadata=metadata, public_access=public_access,
+                                   fail_on_exist=fail_on_exist, timeout=timeout)
```

Comment:
Are the data type of `container is not None` and `client.create_container` consistent?

Reply:
yes